database_scripts/tabels/create_order.sql

```diff
@@ -1,15 +1,18 @@
 use customer_db
 
 
-create table dbo.[Order]
-(
-ID int  NOT NULL PRIMARY KEY 
-CustomerID int FOREIGN KEY dbo.Customer,
-TotalAmount int,
-PaymentType int,
-IsPayed bit, PaymentTypeID int FOREIGN KEY dbo.PaymentType
-[Status] int,
-StatusID int FOREIGN KEY dbo.Status,
-UpdatedDate datetime,
-CreatedDate datetime
-)
\ No newline at end of file
+CREATE TABLE [dbo].[Orders] (
+    [ID] [int] NOT NULL IDENTITY,
+    [CustomerID] [int],
+    [TotalAmount] [decimal](18, 2),
+    [IsPayed] [bit],
+    [PaymentTypeID] [int],
+    [Status] [int],
+    [UpdatedDate] [datetime],
+    [CreatedDate] [datetime],
+    [Status1_ID] [int],
+    CONSTRAINT [PK_dbo.Orders] PRIMARY KEY ([ID])
+)
+CREATE INDEX [IX_CustomerID] ON [dbo].[Orders]([CustomerID])
+CREATE INDEX [IX_PaymentTypeID] ON [dbo].[Orders]([PaymentTypeID])
+CREATE INDEX [IX_Status1_ID] ON [dbo].[Orders]([Status1_ID])
```

Comment:
Tu jest ok, tylko musisz dodać NOT NULL przy polach, które są IsRequired(), długości nvarcharów np. [FirstName] [nvarchar](100), zmień `use customer_db` na `use BProject_db` oraz idexy przenieś do contraintów